homeassistant/components/homekit/type_covers.py

```diff
@@ -135,6 +174,21 @@ def update_state(self, new_state):
         else:
             self.char_position_state.set_value(2)
 
+        # update tilt
+        current_tilt = new_state.attributes.get(ATTR_CURRENT_TILT_POSITION)
+        if isinstance(current_tilt, (float, int)):
+            # HomeKit sends values between -90 and 90.
+            # We'll have to normalize to [0,100]
+            current_tilt = (current_tilt / 100.0 * 180.0) - 90.0
+            current_tilt = int(current_tilt)
+            self.char_current_tilt.set_value(current_tilt)
+
+            if self._homekit_target_tilt is None or abs(
+                current_tilt - self._homekit_target_tilt < 1
+            ):
+                self.char_target_tilt.set_value(current_tilt)
+                self._homekit_target_tilt = None
```

Comment:
@bdraco, based  on [your comment](https://github.com/home-assistant/core/pull/33388#discussion_r399852059
), this should be `< 6` as well. 
I assume it is for when the device reported back a state that is only _close_ to the one that HomeKit requested (because of different precision), and we're "correcting" HomeKit so that it doesn't appear out of sync.

```suggestion
            # We have to assume that the device has worse precision than
            # HomeKit. If it reports back a state that is only _close_ to HK's
            # requested state, we'll "fix" what HomeKit requested.
            DEVICE_PRECISION_LEEWAY = 6
            if self._homekit_target_tilt is None or abs(
                current_tilt - self._homekit_target_tilt < DEVICE_PRECISION_LEEWAY
            ):
                self.char_target_tilt.set_value(current_tilt)
                self._homekit_target_tilt = None
```

Reply:
LGTM.  Would you please move the constant into const.py and use it for the position as well.